src/ontology/fbbt-edit.obo

```diff
@@ -125022,6 +125054,17 @@ relationship: receives_synaptic_input_in_region FBbt:00003888 ! lobula plate lay
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-07-15T10:42:07Z
 
+[Term]
+id: FBbt:00058229
+name: high salt-sensing neuron of the labellum
+def: "A gustatory receptor neuron that innervates a L-type sensillum of the labellum and responds to high concentrations of salts (Liman et al., 2014; Freeman and Dahanukar, 2015)." [FlyBase:FBrf0229299, FlyBase:FBrf0229608]
```

Comment:
They say that S-type bitter-sensing neurons respond *also* to high salts. But this term here (`high salt sensing neuron of the labellum`), if we decide to keep it, is intended for the neurons that respond to high salts *only* (even if weakly) and not to bitter compounds, and those neurons do seem to be found only in L-type sensilla – though it is true that the authors only say it in an indirect manner, by stating that L2 neurons in S- and I-type respond to high salts *and* bitter compounds, silently implying that L2 neurons in L-type sensilla do not have that bitter-sensing ability.

The idea S- and I-type sensilla have neurons with both bitter and high salt sensitivity while L-type sensilla have only high salt-sensing neurons is further advanced by Jaeger et al. (2018; FBrf0240353), which I should have also cited here.

Reply:
Yeah, Jaeger et al. (2018) seems to be the key reference here - I would actually remove the other refs as it is confusing. Also update the label to something more specific - maybe "high salt neuron of L-type sensillum of labellum" or similar, otherwise it sounds like a grouping class.